### PR TITLE
README "first commit" for rancher-utils

### PR DESCRIPTION
**Pull request by koomar**
@siminm 
cc @Nextdoor/engineering-opensource-collaborators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Rancher utilities
+
+## `upgrade_rancher_service.rb`
```
Comment (siminm) [inline README.md]: Either `code` or ##header but not both

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	gem install rancher-api
+	ruby upgrade_rancher_service.rb
+
+#### `circle.yml` example
+
+	deployment:
+	  hub:
+	    branch: master
+	    commands:
+	      - gem install rancher-api; curl 'https://raw.githubusercontent.com/Nextdoor/rancher-utils/master/upgrade_rancher_service.rb' | ruby
```
Comment (siminm) [inline README.md]: Yaml has a multi-line syntax:

```yaml
commands: >
  gem install rancher-api;
  curl url | ruby
```

File: README.md (hunk 1 above)
Comment (koomar) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (koomar) [inline README.md]: Done

**Comment by siminm** [comment]
:+1:

**Pull request merged**